tests/parm/input_global_nest.nml.IN

```diff
@@ -0,0 +1,402 @@
+&atmos_model_nml
+  blocksize = 32
+  chksum_debug = .false.
+  dycore_only = .false.
+  ccpp_suite = "@[CCPP_SUITE]"
+  
+/
+
+&diag_manager_nml
+  prepend_date = .false.
+  max_output_fields = @[MAX_OUTPUT_FIELDS]
+  
+/
+
+&fms_nml
+  clock_grain = 'ROUTINE'
+  domains_stack_size = @[DOMAINS_STACK_SIZE]
+  print_memory_usage = .false.
+  
+/
+
+&fms2_io_nml
+  netcdf_default_format = "netcdf4"
+/
+
+&fv_core_nml
+  layout = @[INPES_NEST02],@[JNPES_NEST02]
+  io_layout = 1,1
+  npx = @[NPX_NEST02]
+  npy = @[NPY_NEST02]
+  ntiles = 1
+  npz = @[NPZ]
+  dz_min =  @[DZ_MIN]
+  psm_bc = @[PSM_BC_NEST02]
+  grid_type = 0
+  make_nh = @[MAKE_NH]
+  fv_debug = .false.
+  range_warn = .true.
+  reset_eta = .false.
+  n_sponge = @[N_SPONGE]
+  n_zs_filter = @[N_ZS_FILTER_NEST02]
+  nudge_qv = .false.
+  nudge_dz = .false.
+  tau = @[FV_CORE_TAU_NEST02]
+  rf_cutoff = @[RF_CUTOFF_NEST02]
+  rf_fast = @[RF_FAST]
+  sg_cutoff = @[SG_CUTOFF]
+  upoff = @[UPOFF]
+  d2_bg_k1 = @[D2_BG_K1_NEST02]
+  d2_bg_k2 = @[D2_BG_K2_NEST02]
+  kord_tm = -9
+  kord_mt = 9
+  kord_wz = 9
+  kord_tr = 9
+  hydrostatic = .false.
+  phys_hydrostatic = .false.
+  use_hydro_pressure = .false.
+  beta = 0.
+  a_imp = 1.
+  p_fac = 0.1
+  k_split = 4
+  n_split = 5
+  nwat = @[NWAT]
+  na_init = @[NA_INIT]
+  d_ext = @[D_EXT]
+  dnats = @[DNATS]
+  fv_sg_adj = @[FV_SG_ADJ]
+  d2_bg = @[D2_BG_NEST02]
+  nord = 2
+  nord_zs_filter = @[NORD_ZS_FILTER]
+  dddmp = @[DDDMP]
+  d4_bg = @[D4_BG_NEST02]
+  vtdm4 = @[VTDM4]
+  delt_max = @[DELT_MAX_NEST02]
+  ke_bg = 0.
+  do_vort_damp = .true.
+  external_ic = @[EXTERNAL_IC]
+  external_eta = .true.
+  gfs_phil = .false.
+  nggps_ic = @[NGGPS_IC]
+  mountain = @[MOUNTAIN]
+  ncep_ic = .false.
+  d_con = @[D_CON]
+  hord_mt = @[HORD_MT_NEST02]
+  hord_vt = @[HORD_VT_NEST02]
+  hord_tm = @[HORD_TM_NEST02]
+  hord_dp = @[HORD_DP_NEST02]
+  hord_tr = @[HORD_TR_NEST02]
+  adjust_dry_mass = .false.
+  dry_mass =  @[DRY_MASS]
+  consv_te = @[CONSV_TE_NEST02]
+  do_sat_adj = @[DO_SAT_ADJ]
+  fast_tau_w_sec = @[FAST_TAU_W_SEC_NEST02]
+  consv_am = .false.
+  fill = .true.
+  dwind_2d = .false.
+  print_freq = 6
+  warm_start = @[WARM_START]
+  no_dycore = .false.
+  z_tracer = .true.
+  agrid_vel_rst = .true.
+  read_increment = @[READ_INCREMENT]
+  res_latlon_dynamics = @[RES_LATLON_DYNAMICS]
+  nested = @[NESTED]
+  twowaynest = @[TWOWAYNEST02]
+  nestupdate = 7
+/
+
+&external_ic_nml
+  filtered_terrain = .true.
+  levp = 128
+  gfs_dwinds = .true.
+  checker_tr = .false.
+  nt_checker = 0
+  
+/
+
+&surf_map_nml
+  cd2 = -1
+  cd4 = 0.15
+  max_slope = 0.15
+  n_del2_strong = @[N_DEL2_STRONG]
+  n_del2_weak = @[N_DEL2_WEAK_NEST02]
+  n_del4 = @[N_DEL4]
+  peak_fac = @[PEAK_FAC]
+  zero_ocean = @[ZERO_OCEAN]
```

Comment:
Lines 122-126:  Does it mean only `n_del2_weak` takes a value different from the parent domain?
(other variables do not have the `_NEST02` suffix)